README.md

```diff
@@ -84,29 +121,41 @@ Changes to be committed:
 	new file:   README.md
 ```
 
-You can now see that Git is ready to keep track of `README.md` and recognizes that the file is new to this repository. However, while we've informed Git that there is a new file, we still haven't informed Git that this new file is considered a change to our repository. We create changes in our repository by making "commits."
+You can now see that Git is ready to keep track of `README.md` and recognizes that the file is new to 
+this repository. However, while we've informed Git that there is a new file, we still haven't informed 
+Git that this new file is considered a change to our repository. We create changes in our repository 
+by making "commits."
 
 **To capture all changes in a directory –– the standard way to do it –– type `git add .`, where the `.` refers to the entire current directory.**
 
-## Cataloging a new version of your code with `git commit`
+## Use `git commit` to apply a message to the commit
```

Comment:
I think this old heading was more informative. Also see previous comment re: inaccurate objective